chromium/background-scripts/background.js

```diff
@@ -147,7 +151,7 @@ function updateState () {
     }
     const tabUrl = new URL(tabs[0].url);
 
-    if (disabledList.has(tabUrl.host) || iconState == "disabled") {
+    if (disabledList.has(tabUrl.host) || httpOnceList.has(tabUrl.host) || iconState == "disabled") {
```

Comment:
Is host guaranteed to be lowercase, punycoded etc?

Reply:
The very nature of the URL JavaScript object doesn't necessarily seem to enforce this ask, but since it grabs the host, which are modeled to be case insensitive, this has been left as is.

Reply:
Do other parts of the code do the same? If yes, this should be fine for now.

Reply:
Yes, the URL object has been utilized this way. Will keep as is for now.